interactions/api/models/guild.py

```diff
@@ -1677,6 +1677,26 @@ async def get_all_channels(self) -> List[Channel]:
         self.channels = [Channel(**channel, _client=self._client) for channel in res]
         return self.channels
 
+    async def get_all_active_threads(self) -> List[Thread]:
```

Comment:
Can we make this return channel? according to https://discord.com/developers/docs/resources/guild#list-active-guild-threads, it is also a list of channels

Reply:
Sure, so we really never use `Thread` object xD